src/gateways/help-request.js

```diff
@@ -1,6 +1,7 @@
 import { DefaultGateway } from '../gateways/default-gateway';
 
-const ToHelpRequest = (hr) => {
+const ToHelpRequestDomain = (hr) => {
+    const helpNeeded = (hr.HelpNeeded == 'Shielding')? 'CEV' : hr.HelpNeeded
```

Comment:
might also be a good idea to do this inline on line 15 (and in the other places you do this)_